src/main/io/gps.c

```diff
@@ -97,6 +97,10 @@ uint8_t GPS_svinfo_cno[GPS_SV_MAXSATS_M8N];
 static serialPort_t *gpsPort;
 static float gpsDataIntervalSeconds;
 
+#ifdef USE_POS_HOLD_MODE
+static bool newDataForPosHold = false;
```

Comment:
(you can use binary counter that increments on each GPS update. Then any code can do:
```
static uint16_t gpsStamp=~0;
if (gpsStamp != gps.stamp) {
    gpsStamp = gps.stamp;
    // got new GPS update
```
Any number of consumers can do this, without creating dependence in GPS code

Reply:
Fantastic!  Thank you!
That's a massively better solution!
I have done it for position hold - will test and if OK will update the other clients also